concepts/bools/about.md

```diff
@@ -27,56 +31,68 @@ True
 
 All `boolean operators` are considered lower precedence than Pythons [`comparison operators`][comparisons], such as `==`, `>`, `<`, `is` and `is not`.
 
+
 ## Type Coercion and Truthiness
 
-The bool function ([`bool()`][bool-function]) converts any object to a Boolean value. By default all objects return `True` unless defined to return `False`.
+The bool function ([`bool()`][bool-function]) converts any object to a Boolean value.
```

Comment:
```suggestion
The `bool` function ([`bool()`][bool-function]) converts any object to a Boolean value.
```

Because that is a function name, and code, so code "markuped".  Also, reinforcing my prior mention of capitalization of the word "Boolean", it is done here, and so the prior uncapitalized name seems out of place even more.